### DISCUSSION TITLE
【已解决】宝塔面板搭建访问不了

### DISCUSSION THREAD

**Discussion by 1834423612** [AlistGo/alist#1353] [Q&A]
按照官方文档使用一键安装，已确认安装成功，且使用ip访问正常，但按照官方文档的宝塔设置反代理出错如图二、图三，访问时一直在加载页面，且弹出的红框没有任何文字提示，请大佬看看该如何解决？
![image](https://user-images.githubusercontent.com/49981661/179554067-983a9854-4bde-465d-b4ed-e3cc0568fce1.png)

![image](https://user-images.githubusercontent.com/49981661/179554331-d0b58aca-0622-43f7-8f7a-d511f4f38f63.png)
![image](https://user-images.githubusercontent.com/49981661/179554419-176a2bdb-749a-4d5f-83a3-7666e877fb92.png)

**Comment by Becods** [comment]
![CM6PHS7AGGJPN31Y2DI83MS](https://user-images.githubusercontent.com/35525491/179665759-169daf92-600a-49e6-a99d-af49420632ed.png)

**Comment by 1834423612** [comment]
已解决，alist宝塔添加站点只能使用一个域名，多个域名占用就会出现白屏。（压根就没有错误日志😶）
